pkg/testutil/env/sandbox.go

```diff
@@ -172,20 +175,16 @@ func createServiceAccount(s Sandbox) error {
 }
 
 func startCtrlMgr(t *testing.T, mgr manager.Manager) func() {
-	stop := make(chan struct{})
-	wg := &sync.WaitGroup{}
-	wg.Add(1)
+	ctx, cancel := context.WithCancel(context.Background())
 
 	go func() {
-		defer wg.Done()
-		if err := mgr.Start(stop); err != nil {
+		if err := mgr.Start(ctx); err != nil {
 			t.Fatal(err)
 		}
 	}()
 
 	return func() {
```

Comment:
nit: `return cancel` is a bit more clear

Reply:
Does the way I refactored it look correct?